src/libsystemd/sd-varlink/sd-varlink.c

```diff
@@ -1061,23 +1061,23 @@ static int varlink_dispatch_disconnect(sd_varlink *v) {
         return 1;
 }
 
+
 static int varlink_sanitize_parameters(sd_json_variant **v) {
         int r;
-
```

Comment:
separating empty line dropped?

(we usually place a separating line between variable decls and func calls)